sql/1692105853.sql

```diff
@@ -0,0 +1,76 @@
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS convoy.events_search
+(
+
+    id                 VARCHAR NOT NULL PRIMARY KEY,
+    event_type         TEXT    NOT NULL,
+    endpoints          TEXT,
+    project_id         VARCHAR NOT NULL REFERENCES convoy.projects (id),
+    source_id          VARCHAR REFERENCES convoy.sources (id),
+    headers            JSONB,
+    raw                TEXT    NOT NULL,
+    data               BYTEA   NOT NULL,
+    url_query_params   VARCHAR,
+    idempotency_key    TEXT,
+    is_duplicate_event BOOLEAN DEFAULT FALSE,
+    search_token       TSVECTOR GENERATED ALWAYS AS (to_tsvector('simple', raw)) STORED,
+    created_at         TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
+    updated_at         TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
+    deleted_at         TIMESTAMP WITH TIME ZONE
+);
```

Comment:
Do we need all the fields in events_search?

Reply:
Probably, I don't want to have to make two queries.